content/en/tracing/guide/aws_payload_tagging.md

```diff
@@ -147,26 +148,27 @@ The Node.js tracer applies redaction rules on a per-service basis. For example:
 {{< programming-lang lang="python" >}}
 ### Supported services
 
-The following services are supported:
+The following services are supported by default:
 - SNS
 - SQS
 - Kinesis
 - S3
 - EventBridge
+- DynamoDB
 
 #### Configure services
 
 To enable tag extraction for additional services, use this environment variable:
 
 ```sh
 # Default values
-DD_TRACE_CLOUD_PAYLOAD_TAGGING_SERVICES=s3,sns,sqs,kinesis,eventbridge
+DD_TRACE_CLOUD_PAYLOAD_TAGGING_SERVICES=s3,sns,sqs,kinesis,eventbridge,dynamodb
```

Comment:
This is currently not the case. But will be like this after[ this PR](https://github.com/DataDog/dd-trace-py/pull/12688/files)

Reply:
I'll remove it for now